wandb/cli/cli.py

```diff
@@ -1349,10 +1356,15 @@ def launch_agent(
     queues=None,
     max_jobs=None,
     config=None,
+    url=None,
 ):
     logger.info(
         f"=== Launch-agent called with kwargs {locals()}  CLI Version: {wandb.__version__} ==="
     )
+    if url is not None:
+        raise LaunchError(
+            "--url is not supported in this version, upgrade with: pip install -u wandb"
```

Comment:
Could we put a link to a `wandb.me` url that outlines this feature rather than a hard coded error message? Users trying out launch could be confused by this in the upcoming public preview roll out

Reply:
As discussed we believe since this is hidden no one will ever hit this until we actually implement the feature at which point upgrading will be the solution.